internal/adapters/terraform/aws/elasticsearch/adapt.go

```diff
@@ -24,11 +24,15 @@ func adaptDomains(modules terraform.Modules) []elasticsearch.Domain {
 
 func adaptDomain(resource *terraform.Block) elasticsearch.Domain {
 	domain := elasticsearch.Domain{
-		Metadata:   resource.GetMetadata(),
-		DomainName: defsecTypes.StringDefault("", resource.GetMetadata()),
+		Metadata:               resource.GetMetadata(),
+		DomainName:             defsecTypes.StringDefault("", resource.GetMetadata()),
+		AccessPolicies:         resource.GetAttribute("access_policies").AsStringValueOrDefault("", resource),
+		VpcId:                  resource.GetAttribute("vpc_options.0.vpc_id").AsStringValueOrDefault("", resource),
```

Comment:
how did you get these attributes? What if there's more than one VPC option? I assume `vpc_options.0` only supports the first?

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elasticsearch_domain...I follow this documentation

Reply:
perfect